server/app.js

```diff
@@ -472,9 +472,18 @@ const getPureError = (error)=>{
 };
 
 app.use(async (err, req, res, next)=>{
-	const status = err.status || err.code || 500;
+	err.originalUrl = req.originalUrl;
 	console.error(err);
 
+	if(err.originalUrl?.startsWith('/api/')) {
+		// console.log('API error');
+		res.status(err?.status || err?.response?.status || 500).send(err.message || err);

```

Comment:
Do you need the `?` operator on `err?.status` and `err?.response`?  Presumably there *is* an error if you've entered this `if...then` statement.  

If anything, they could be replaced by throwing the `?` in the `if(err?.originalURL....` condition?

Reply:
I think that entire line came directly from `homebrew.api.js`, you are correct - there is no way that we can enter that branch of the `if` with a non-existent `err`.